frontend/src/services/Fee/utxo/FeeServiceUtxo.ts

```diff
@@ -1,11 +1,20 @@
 /* eslint-disable */
 import { Coin } from '../../../gen/vultisig/keysign/v1/coin_pb';
 import { SpecificUtxo } from '../../../model/gas-info';
+import { RpcServiceFactory } from '../../Rpc/RpcServiceFactory';
 import { FeeService } from '../FeeService';
 import { IFeeService } from '../IFeeService';
 
 export class FeeServiceUtxo extends FeeService implements IFeeService {
   async getFee(coin: Coin): Promise<SpecificUtxo> {
-    return {} as SpecificUtxo;
+    const rpcService = RpcServiceFactory.createRpcService(this.chain);
+    try {
+      let gasInfo = (await rpcService.getGasInfo(coin)) as SpecificUtxo;
+      gasInfo.gasPrice = gasInfo.gasPrice;
```

Comment:
what is this line for?

Reply:
Usually we need to do some transformation, per chain/coin so we do it here. 

Example:
https://github.com/vultisig/vultisig-windows/blob/doge/frontend/src/services/Fee/evm/FeeServiceEvm.ts#L14

Reply:
Removed. Not needed in this case.